config/example/admins.txt

```diff
@@ -135,3 +135,4 @@ Naksuasdf = Game Master
 MrDoomBringer = Game Master
 shizcalev = Game Master
 NicBR = Game Master
+Wilchen = Host
```

Comment:
speedmerge

Reply:
configs are serverside and this is merely so i can test this on my end

Reply:
just makin' the joke, don't worry. did you figure out what the issue was last time?

Reply:
ok